.jenkins/actions/run_standalone.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# Jenkins action to run a benchmark of dynamics.py on Piz Daint
+# 3/11/2021, Tobias Wicky, Vulcan Inc
+
+# stop on all errors and echo commands
+set -e
+
+# utility function for error handling
+exitError()
+{
+    echo "ERROR $1: $3" 1>&2
+    echo "ERROR     LOCATION=$0" 1>&2
+    echo "ERROR     LINE=$2" 1>&2
+    exit $1
+}
+
+# check arguments
```

Comment:
do we have a convention for describing the arguments in a starting comment block?

It usually helps me personally (how often did I have to check if ranks or time steps was the first argument...)